app/serializers/framework_assessment_serializer.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class FrameworkAssessmentSerializer
+  include JSONAPI::Serializer
+
+  belongs_to :framework
+
+  has_many :responses, serializer: FrameworkResponseSerializer do |object|
+    object.framework_responses.includes(:framework_flags, :framework_nomis_mappings, framework_question: [:framework, dependents: :dependents])
+  end
+  has_many :flags, serializer: FrameworkFlagSerializer do |object|
+    object.framework_flags.includes(framework_question: :dependents)
+  end
+
+  attributes :confirmed_at, :created_at, :nomis_sync_status
+
+  attribute :version do |object|
+    object.framework.version
+  end
+
+  attribute :status do |object|
+    object.status == 'unstarted' ? 'not_started' : object.status
```

Comment:
I'm sure there is a story behind this mapping... ...but is it possible to rename `unstarted` to `not_started` instead?

Reply:
Unfortunately due to enum values throwing a warning if their name is prepended with a "not_", we kept the enum name as "unstarted" however it doesn't read well being surfaced, so we return "not_started" instead. I've added a comment to avoid confusion.

Reply:
👍